apps/approval/tests.py

```diff
@@ -97,6 +99,7 @@ def setUp(self):
         self.logger = logging.getLogger(__name__)
 
     def testEmailWhenMembershipDenied(self):
+        G(Permission, permission_type=PermissionType.APPLICATIONS, force_email=True)
```

Comment:
Why is force_email set to True in for the tests here?

Reply:
We use force_email to override any user define permissions for mail. We would define force_email for all permission types that require us to send mail

Reply:
Oh right, so it is to isolate the test, as we do not want to test notification permissions here, just that emails are sent on membership denied?

Reply:
Yes, exactly that :smile: